README.md

```diff
@@ -51,14 +51,14 @@ for examples of the packages required for several common Linux distributions.
 <!--If you change the name of this header, please update all links to it as well-->
 
 Clone the [`hse`](https://github.com/hse-project/hse) repo and checkout
-the latest release tag.  This tag must be for HSE version 2.0 or higher.
+the latest release tag. This tag must be for HSE version 2.0 or higher.
 
-For example
+For example:
 
 ```shell
 git clone https://github.com/hse-project/hse.git
 cd hse
-git checkout <release tag>
+git checkout $releaseTag
```

Comment:
The <something> format seems more common than the $something format.  I don't feel strongly either way, but we should be consistent and use  the same format in our README.md in hse-mongo and hse-ycsb (and probably elsewhere).

Reply:
Only reason I changed it was to get the syntax highlighting to look better when viewed on GitHub. Happy to revert.

Reply:
Alternatively I could just remove the syntax highlighting from this snippet.

Reply:
It's not highlighting, but rather a common format used in docs to represent a placeholder, in this case a placeholder for the specific release tag to be checked-out.  Other common placeholder formats (e.g., from man pages) use unadorned caps (RELEASE_TAG) or unadorned lower-case italics (*release_tag*). 

I have no preference, other than that we be consistent.